webhooks-extension/config/triggers_prototype/ingress.yaml

```diff
@@ -28,6 +31,106 @@ spec:
       default: "8082"
       type: string
   steps:
+  - name: generate-certificate
+    image: cfssl/cfssl
```

Comment:
Changes look good aside from export clearance stuff for shipping this (used in a Tekton Task I know) for products, I don't think this changes how Duane's code works (but we can then tidy up the readme and say you either use this task, or it's done through code, *or* you use the original "bring your own cert method)

I see this as a much better image to use than the non-official openssl images though 👍

Reply:
@a-roberts What do I need for this?  Export clearance?

Reply:
Well I was first concerned around using OpenSSL in a Tekton task as it does have encryption capabilities I reckon (and one can get in biiiig trouble for not following the procedures), and because our goodies make their way into products, it's not clear on how we should proceed from a legal perspective really. 

Everything else is great, it's just this bit we should clear up (perhaps leveraging IBM legal teams and management and this is of use to anybody else that wants to use this task too - because you would end up with that image on your machine and there are procedures to follow)